api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md

```diff
@@ -61,9 +61,9 @@ Gets and sets the delay between when a user stops typing the field value and whe
 The editor's width.
 
```

Comment:
Can you update line 55 above?

Line 55:
`Gets and sets the delay between when a user stops typing the field value and when it is applied.`

Can we make "it" more specific?

`Gets and sets the delay between when a user stops typing the field value and when the change is applied.`

Reply:
Noted

Reply:
Thanks.